### DISCUSSION TITLE
How to pragmatically navigate

### DISCUSSION THREAD

**Discussion by kresli** [TanStack/router#109] [Q&A] [answered]
Is there a way how to push a new path pragmatically?
something like
```
const navigation = useNavigation();
navigation.push("/lobin");
```

**Comment by kresli** [comment, marked as answer]
found it https://react-location.tanstack.com/docs/api#usenavigate

Seems like the search function in docs doesn't work at all.

**Reply by tannerlinsley** [reply]
Are you sure? useNavigate works for all of the examples. Please provide a code sandbox to reproduce.

**Reply by kresli** [reply]
Sorry, what I meant the search in https://react-location.tanstack.com/ doesn't work and that's why I couldn't find useNavigate method. All good now
![image](https://user-images.githubusercontent.com/3826097/142780740-0b90ce6d-884b-438e-8747-87dbeac2d641.png)
